companion/gateway/routers/admin.py

```
@@ -0,0 +1,368 @@
+            "timestamp": datetime.utcnow()
+        }
+    except Exception as e:
+        logger.error(f"Error getting scheduler status: {e}")
+        raise HTTPException(status_code=500, detail="Internal server error")
+
+
+@router.post("/scheduler/{job_id}/control", response_model=Dict[str, str])
+async def control_scheduler_job(
+    job_id: str,
+    action: str = Query(..., regex="^(pause|resume|trigger)$"),
+    user_service: UserService = Depends(get_users)
```

Ruff (0.13.2)

Unused function argument: `user_service`

(ARG001)